### DISCUSSION TITLE
Placebo on the brain

### DISCUSSION THREAD

**Discussion by mariadigitalisera** [NeurodataWithoutBorders/helpdesk#26] [Ideas]
Hi guys I am preparing a week long workshop about placebo as part of brain awareness week 14-20th of March would love to get
some feedback on material that I am preparing before 14th of March would anyone here be interested to give me some feedback if I put up a link to a demo on 6th of March?
I can be reached at maria.lindqvist.9382@student.uu.se I am also on slack

**Comment by bendichter** [comment]
Hi @mariadigitalisera, were you planning on using NWB data for this workshop? Or convert existing data into NWB? I do not know off-hand of any datasets on the [DANDI Archive](dandiarchive.org) for instance that contain placebo data, but would be happy to help convert data if that is your plan.

**Comment by mariadigitalisera** [comment]
Thank you for the note! I will get back with all
details on 6th of March!

Best regards;

Maria
________________________________
Från: Ben Dichter ***@***.***>
Skickat: den 27 februari 2022 00:57
Till: NeurodataWithoutBorders/helpdesk ***@***.***>
Kopia: Maria Lindqvist ***@***.***>; Mention ***@***.***>
Ämne: Re: [NeurodataWithoutBorders/helpdesk] Placebo on the brain (Discussion #26)


Hi @mariadigitalisera<https://github.com/mariadigitalisera>, were you planning on using NWB data for this workshop? Or convert existing data into NWB? I do not know off-hand of any datasets on the DANDI Archive<dandiarchive.org> for instance that contain placebo data, but would be happy to help convert data if that is your plan.

—
Reply to this email directly, view it on GitHub<https://github.com/NeurodataWithoutBorders/helpdesk/discussions/26#discussioncomment-2258625>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APR3GE7N5TJDWUAXR45AXOLU5FSH3ANCNFSM5PNDZVEQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
